Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/calltree/CallTreeProvider.java

```diff
@@ -283,7 +283,7 @@ public boolean isAddToPopup(ActionContext context) {
 
 					for (TreePath path : selectionPaths) {
 						GTreeNode node = (GTreeNode) path.getLastPathComponent();
-						if (node instanceof GTreeNode) {
+						if (node != null) {
 							return false;
 						}
```

Comment:
Hm I didn't dig that deep into the code, I just realized that this check is useless and only might fail if `node == null`

Reply:
Actually I'm a bit confused right now about the `isAddToPopup`-method... I think it will always `return false`, because it would only `return true`, if selectionPaths is not empty and contains no path which has a lastComponent which is null, which per contract I think should not be possible. So actually this whole method just always returns true if I'm not mistaken... And I also don't see any side-effects here, so probably this can just be replaced with `return false` 😕 Is this correct?